extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -1129,9 +870,22 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
       // Allow non-AMP creatives to remain unless SRA.
       return false;
     }
-    this.binding_.unlayoutCallback();
-    const superResult = super.unlayoutCallback();
-    return superResult;
+    this.destroySafeFrameApi_();
+    return super.unlayoutCallback();
+  }
+
+  /** @visibleForTesting */
+  cleanupAfterTest() {
+    this.destroySafeFrameApi_();
+  }
+
+  /** @private */
```

Comment:
can you not just make this directly visibleForTesting?

Reply:
I'd rather expose a private method so it can be used on `unlayoutCallback`. Two entry points :)